packages/opentelemetry-metrics/src/export/Aggregator.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Distribution, Sum, LastValue } from './types';
+import { hrTime } from '@opentelemetry/core';
+import * as types from '@opentelemetry/types';
+
+/**
+ * Base interface for aggregators. Aggregators are responsible for holding
+ * aggregated values and taking a snapshot of these values upon export.
+ */
+export interface Aggregator {
```

Comment:
Does this belong in `./types' file?

Reply:
?

Reply:
done in 5469514c16e35779da28564e56f597823ad3623b